userguide/virtualmachines.rst

```diff
@@ -663,107 +663,88 @@ Start the Docker Host
+   Docker Host.
 
 
-If the terminal does not show a :literal:`rancher login:` prompt,
-press :kbd:`Enter`. The Docker Host can take some time to start and
-display the login prompt.
+When a message about :literal:`RancherOS` starting appears and the shell
+stops posting new messages, press :kbd:`Enter` to see the
+:literal:`ClientHost login:` text and continue to log in.
 
 
 .. _Installing and Configuring the Rancher Server:
 
 Installing and Configuring the Rancher Server
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Go to
-:menuselection:`Virtual Machines`
-and locate the card for the Docker Host. Start the Host and click
-:guilabel:`Connect` and :guilabel:`Serial` to open the Host Serial shell.
-
-Continuing to set up the Rancher Host is done from the command line.
-Enter *rancher* as the username, press :kbd:`Enter`, then enter either
-the default password *docker* or the custom password created by editing
-the raw file. Press :kbd:`Enter` again. After logging in, a
-:literal:`[rancher@ClientHost ~]$` prompt is displayed.
+In the serial shell, type :literal:`rancher` for the username and press
+:kbd:`Enter`. Enter either the default password :literal:`docker` or the
+custom password created by editing the raw file and press :kbd:`Enter`.
+A :literal:`[rancher@ClientHost ~]$` prompt will appear.
 
 Ensure Rancher has functional networking and can :command:`ping` an
-outside website. Adjust the VM
-:ref:`Network Interface <vms-network-interface>` and reboot the VM
-if necessary.
-
-Download and install the Rancher system with this command:
+outside website.
 
 .. code-block:: none
 
-   sudo docker run -d --restart=unless-stopped -p 8080:8080 rancher/server
+   [rancher@ClientHost ~]$ ping -c 5 8.8.8.8
+   PING 8.8.8.8 (8.8.8.8): 56 data bytes
+   64 bytes from 8.8.8.8: seq=0 ttl=121 time=17.466 ms
+   64 bytes from 8.8.8.8: seq=1 ttl=121 time=17.502 ms
+   64 bytes from 8.8.8.8: seq=2 ttl=121 time=17.472 ms
+   64 bytes from 8.8.8.8: seq=3 ttl=121 time=17.426 ms
+   64 bytes from 8.8.8.8: seq=4 ttl=121 time=17.343 ms
 
+   --- 8.8.8.8 ping statistics ---
+   5 packets transmitted, 5 packets received, 0% packet loss
+   round-trip min/avg/max = 17.343/17.441/17.502 ms
 
-.. note:: If the error :literal:`Cannot connect to the Docker daemon`
-   is shown, run :command:`sudo dockerd`. Then give the
-   :command:`sudo docker run` command above again.
 
+If :command:`ping` returns an error, adjust the VM
+:ref:`Network Interface <vms-network-interface>` and reboot the VM.
 
-Installation time varies with processor and network connection speed,
-but typically takes a few minutes. After the process finishes and a
-command prompt is shown, type this command:
-
-.. code-block:: none
-
-   ifconfig eth0 | grep 'inet addr'
+Download and install the Rancher system with
+:command:`sudo docker run -d --restart=unless-stopped -p 8080:8080 rancher/server`.
 
+If :literal:`Cannot connect to the Docker daemon` shows, run
```

Comment:
If/then, although probably okay here.  But it sounds unsure.
"is shown" works better than "shows" here, but probably better to explicitly point out this is an error.
Also, "run" is oddly used.
"If a :literal:`Cannot connect to the Docker daemon` error is shown, enter"